v-next/core/src/internal/utils/package.ts

```diff
@@ -0,0 +1,19 @@
+import type { PackageJson } from "@ignored/hardhat-vnext-utils/package";
```

Comment:
This should also live in hardhat, not core

Reply:
Done [here](https://github.com/NomicFoundation/hardhat/pull/5580/commits/8f13be53096fc3eea0191fef0b4b28e3312124ce)